packages/erxes-ui/src/components/richTextEditor/RichTextEditorControl/MoreButtonControl.tsx

```diff
@@ -38,7 +38,7 @@ export const MoreButtonControl = (props: IRichTextEditorMoreControlProps) => {
 
   return (
     <OverlayTrigger
-      rootClose={true}
+      rootClose={false}
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (code_clarification):** Review the 'rootClose' prop change in MoreButtonControl.

The change in 'rootClose' prop from true to false in MoreButtonControl might affect the expected behavior. Ensure this change is intentional and tested.